tests/Doctrine/Tests/DBAL/Functional/ResultCacheTest.php

```diff
@@ -166,6 +166,21 @@ public function testDontFinishNoCache()
         self::assertCount(2, $this->sqlLogger->queries);
     }
 
+    public function testFetchAllAndFinishSavesCache()
+    {
+        $layerCache = new ArrayCache();
+        $stmt = $this->connection->executeQuery('SELECT * FROM caching WHERE test_int > 500', [], [], new QueryCacheProfile(10, 'testcachekey', $layerCache));
+        $stmt->fetchAll();
+        $stmt->closeCursor();
+
+        $cachedData = $layerCache->fetch('testcachekey');
+        self::assertNotFalse($cachedData);
```

Comment:
This is redundant. The assertion below is enough.

Reply:
Thanks, deleted.